cmd/gofish/ensure_unix.go

```diff
@@ -10,11 +10,24 @@ import (
 )
 
 func ensureDirectories(dirs []string) error {
+	var isRoot bool = false
+
 	userCmd := exec.Command("id", "-un")
 	userOutput, err := userCmd.Output()
 	if err != nil {
 		return err
 	}
+
+	userCmd = exec.Command("id", "-u")
```

Comment:
Have you considered using `user.Current` instead?

Reply:
https://pkg.go.dev/os/user@go1.17#Current

Reply:
switched to user.Current.